File: fizzbizzbuzz.py
def FizzBuzz (start, stop):
    if type (start)!= int and type (stop) != int :
        print ("not an interger") 
    
    for fizzbuzz in range(start,stop):
        if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
           print("FizzBuzz")
           continue
        elif fizzbuzz % 3 == 0:
           print("Fizz")
           continue
        elif fizzbuzz % 5 == 0:
           print("Buzz")
           continue
        print(fizzbuzz)
FizzBuzz("3000","7000")

File: fizzbizzbuzz.js
function fizzbuzz(val){

    if(typeof val !== "number") {
	console.log("Must be a number!");
	return;
}

   for(let i = 1; i <= val ; i++){
    if(i % 3 === 0 && i % 5 === 0){
     console.log("FizzBuzz");
    }else if (i % 3 === 0){
     console.log("Fizz");
    }else if (i % 5 === 0){
     console.log("Buzz");
    }else{
     console.log(i);
    }
  }
}

fizzbuzz(7)